app/src/main/java/com/example/nidoqueue/controller/DatabaseManager.java

```diff
@@ -0,0 +1,125 @@
+package com.example.nidoqueue.controller;
+
```

Comment:
I want to put the DatabaseManager in the Model layer, so that it's the model layer that is updating the view.  That way we are technically doing MVC properly.

Reply:
I see

Reply:
But how can we instantiate this only once? instantiate in Singleton class? It is a waste of resources if we instantiate this every single time

Reply:
Im pretty sure there is only one instance of a Singleton.  Every time a file calls getInstance(), I'm pretty sure the program doesn't need to store anything else in memory except maybe a pointer to the instance.

Reply:
https://github.com/CMPUT301W21T19/Nidoqueue/pull/67#discussion_r602982705